rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -197,61 +172,59 @@ void SetupResumeAcceptor::accept(
   auto processor = std::make_shared<OneFrameProcessor>(
       *this, transport, std::move(onSetup), std::move(onResume));
   connections_.insert(transport);
-  // transport can receive frames right away
+  // Transport can receive frames right away.
   transport->setFrameProcessor(std::move(processor));
 }
 
-std::shared_ptr<FrameSerializer>
-SetupResumeAcceptor::getOrAutodetectFrameSerializer(
-    const folly::IOBuf& firstFrame) {
-  if (defaultFrameSerializer_) {
-    return defaultFrameSerializer_;
+std::shared_ptr<FrameSerializer> SetupResumeAcceptor::frameSerializer(
```

Comment:
I would suggest to add the word "detect" in the name of the method. 
frameSerializer() doesn't suggest its an "expensive method" doing some heuristic to return FrameSerializer.